src/buttons/Button.js

```diff
@@ -50,9 +52,23 @@ class Button extends Component {
       iconContainerStyle,
       iconRight,
       linearGradientProps,
+      disabled,
+      disabledStyle,
+      disabledTextStyle,
       ...attributes
     } = this.props;
 
+    // this is what RN Button does by default
+    // https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js#L118
+    const Touchable =
+      Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+
+    // FIXME: This doesn't work for non-expo users. `require('expo')` is evaluated anyway.
+    // const ButtonContainer = linearGradientProps
+    //   ? require('expo').LinearGradient
+    //   : View;
+    const ButtonContainer = View;
```

Comment:
These lines (61-70) are useless, since `TouchableComponent` and `ViewComponent` are already doing the job (see `defaultProps`).